addons/mrp/tests/test_warehouse_multistep_manufacturing.py

```diff
@@ -810,3 +810,28 @@ def test_manufacturing_bom_with_repetitions(self):
         mo_P2 = self.env['mrp.production'].search([('product_id', '=', products[2].id)])
         self.assertEqual(mo_P1.product_uom_qty, 5.0)
         self.assertEqual(mo_P2.product_uom_qty, 5.0)
+
+    def test_update_component_qty(self):
+        warehouse = self.warehouse
+        warehouse.manufacture_steps = "pbm"
+        bom = self.bom
+        product = bom.product_id
```

Comment:
```suggestion
        self.warehouse.manufacture_steps = "pbm"
```
Since they're all used once anyway and don't shorten the expressions much anyway, I think we can skip those lines. :slightly_smiling_face: